src/main/scala/nl/biopet/tools/bamstats/generate/Args.scala

```diff
@@ -25,9 +25,8 @@ import java.io.File
 
 case class Args(outputDir: File = null,
                 bamFile: File = null,
-                sample: String = null,
-                library: String = null,
-                readgroup: String = null,
+                defaultSample: Option[String] = None,
```

Comment:
I think we can be more strict here. If a readgroup exists without sample or library an exception may be trown.
This should be mentioned in the docs / log that this can be fixed with picard AddOrReplaceReadgroups.